script/Ciel/Day 11/11_01_CIEL11_1.txt

```diff
@@ -120,142 +121,144 @@ No, that's not right.
 -- Page 5, Offset 28994.
 -- この両手は、倒れている子供の血で赤いだけ。
--- TRANSLATION HERE
+My hands are red from the collapsed child's blood.
 }
 [sha:5cf5b2cd7a28e578dbc25eca0ca4fc76251ff4c0]{
 -- Page 6, Offset 28995.
 -- つまりオレは。
--- TRANSLATION HERE
+Which means...
 }
 [sha:da4b36ab2c20a43e5df90ee04874cd37ed10b9c0]{
 -- Page 6, Offset 28996.
 -- ガキの時分から、人でなしの人殺しだった。
--- TRANSLATION HERE
+I've been a despicable murderer since I was a kid.
 }
 [sha:0be0a14ad349bdf20f2438ac0c117024e0546167]{
 -- Page 7, Offset 28997.
 -- 日の当たらぬ庭園。
--- TRANSLATION HERE
+It is a garden which the sun does not reach.
 }
 [sha:f59b093ad8d5cc2754b1f2aea1d4d9c29a1cd42f]{
 -- Page 7, Offset 28998.
 -- 月光だけを頼りに咲き誇る大輪の花々。
--- TRANSLATION HERE
+Where full flowers bloom under the guidance of nothing but moonlight.
 }
 [sha:14969d460fbf6d69e0ad1fd6a2d24fed105e970a]{
 -- Page 7, Offset 28999.
 -- 天には<宙|ソラ>を塞ぐ<白貌|はくぼう>。
--- TRANSLATION HERE
+In the heavens hangs the white face which seals off the cosmos 
 }
 [sha:265905fdad7c17837f09caaedb8cb7d260ad7669]{
 -- Page 7, Offset 29000.
 -- 巡る星の<最中|さなか>には無垢純白の女がひとり。
--- TRANSLATION HERE
+In the midst of the streaking stars stands a lone unblemished woman.
 }
 [sha:737eb7b2c2852958977b8752641ff0d6e65df510]{
 -- Page 8, Offset 29001.
 -- 言葉さえ知らず。
--- TRANSLATION HERE
+She knows not words
 }
 [sha:817b5cb07b85702bc17f57f5d045f61322af2dc9]{
 -- Page 8, Offset 29002.
 -- 自己の意義も解せず。
--- TRANSLATION HERE
+She grasps not her own significance.
 }
 [sha:9f7a40cab4f37d63a1eca9c5da52b859f090b5fa]{
 -- Page 8, Offset 29003.
 -- 殺戮の手段としてしか扱われない。
--- TRANSLATION HERE
+She is only ever used as a method of massacre.
 }
 [sha:cd1a3d1b0c088f6eb59dfcf6b2721fc568462854]{
 -- Page 8, Offset 29004.
 -- 鮮血にまみれながら傷はなく。
--- TRANSLATION HERE
+Drenched in blood, yet without wounds.
 }
 [sha:8024a238b69af33538a8742eea255558fb87f0f9]{
 -- Page 8, Offset 29005.
 -- ドレスを濡らす朱は<同胞|はらから>どもの心血だけ。
--- TRANSLATION HERE
+The only crimson that may stain her dress is the lifeblood of her brethren.
 }
 [sha:b54f02afdc72543f0e1d4607bb1eabeb937ddb54]{
 -- Page 9, Offset 29006.
 -- 女に許された自由はそれだけ。
--- TRANSLATION HERE
+This is the only freedom allowed to her.
 }
 [sha:3ee4c6acb315d5fce8a564d011e605fa186267c4]{
 -- Page 9, Offset 29007.
 -- 用を済ませ帰還した女は、落ちる星のように城の深部に幽閉される。
--- TRANSLATION HERE
+When she has completed her task, she returns to the castle like a falling star, to be imprisoned in its furthest depths
 }
 [sha:0726bc472df427ca1bc22650745aa97230753c9d]{
 -- Page 9, Offset 29008.
 -- 自身で起きる事さえできない眠りに落ちる。
--- TRANSLATION HERE
+She falls into slumber, unable to even wake up on her own.
 }
 [sha:4edf9b16d96fb6ad911f66ecc7a0237e1c622689]{
 -- Page 9, Offset 29009.
 -- ……白い女はその運命さえ知らず、ただ、遠い目で月を見上げている。
--- TRANSLATION HERE
+Yet the woman in white in unaware of even that fate, merely gazing upon the Moon, her eyes set unto the distance.
 }
 [sha:924823c4f1b09c53931d1c10ea110abf31357898]{
 -- Page 10, Offset 29010.
 -- ―――そこに、永遠を見た、と思った。
--- TRANSLATION HERE
+...There, I think I saw eternity.
```

Comment:
is this past tense promoted intentionally? I thought he's describing what he thought and then musing on it in the present tense.

Reply:
For some reason  I wrote the first part of this scene in present tense, and the second in past tense (when it goes from being a description to Roa's narration)

I'm not sure if its appropriate, but it felt natural to write...